wasmplugin/plugin.go

```diff
@@ -54,29 +54,24 @@ func (m *wafMap) put(key string, waf coraza.WAF) error {
 	return nil
 }
 
-func (m *wafMap) setDefaultKey(key string) error {
-	if len(key) == 0 {
-		return errors.New("empty default WAF key")
-	}
-
-	if _, ok := m.kv[key]; ok {
-		m.defaultKey = key
-		return nil
+func (m *wafMap) setDefaultWAF(w coraza.WAF) error {
+	if w == nil {
+		return errors.New("nil WAF set as default")
```

Comment:
Just noticed, these nil checks seem like programming bugs so should be panics, not error return

Reply:
Done.